src/nl/rubensten/texifyidea/settings/TexifySettings.kt

```diff
@@ -24,6 +24,9 @@ class TexifySettings : PersistentStateComponent<TexifySettings> {
     var automaticUpDownBracket = true
     var automaticItemInItemize = true
 
+    // Index of selected combobox item
+    var automaticQuoteReplacement = 0
```

Comment:
It'd better to use an enum to represent all states and use the enum elements for the combo box. Now it's some non-sensible magic value.

Reply:
Makes sense, fixed that.